### PR TITLE
Improve Filtering of Swift Symbols

### PR DESCRIPTION
**Pull request by Br1an-Boyle**
- If an Objective-C class has the title `Swift` in it e.g `SentrySwiftAsyncIntegration` then it was being excluded from the list of symbols to mangle.
- Added improved Swift Symbol filtering so that we only target actual Swift symbols

### CODE REVIEW

File: lib/cocoapods_mangle/defines.rb [hunk 1]
```diff
@@ -125,9 +125,15 @@ def self.swift_symbol?(symbol)
...
-        # Internal Swift symbols starting with Swift such as should not be mangled
-        # e.g. 'Swift51Override' 
-        symbol[/Swift/] ||
+        # Internal Swift symbols starting with digit+Swift+optional_digit should not be mangled
+        # e.g. '34SwiftOverride', '34Swift570Override' 
+        symbol[/\d+Swift(\d+)?/] ||
+        # Internal Swift symbols starting with Swift+digit should not be mangled
+        # e.g. 'Swift570Override' 
+        symbol[/Swift\d+/] ||        
+        # Internal SwiftUI symbols starting with digit+SwiftU+optional_digit such as should not be mangled
```
Comment (mapierce) [inline lib/cocoapods_mangle/defines.rb:134]: Typo here, I think it should be `SwiftUI+optional_digit`

File: spec/unit/defines_spec.rb [hunk 2]
```diff
@@ -35,6 +35,7 @@
         FLAnimatedImage
         FLWeakProxy
         FLAnimatedImageView
+        BrianSwiftAsyncIntegration
```
Comment (mapierce) [inline spec/unit/defines_spec.rb:38]: Can we add a test for SwiftUI mangling since you made a change to include it?

Comment (Br1an-Boyle) [reply spec/unit/defines_spec.rb:38]: Yep. I've added `SwiftUI` to the symbols text file. The idea is that this _isn't_ included in the mangling. So, we don't want to add an additional check here.

Comment (mapierce) [reply spec/unit/defines_spec.rb:38]: Gotcha. Thanks!

**Review by mapierce — APPROVED** [review]

**Pull request merged**